### ISSUE TITLE
Showtitlebar can not fill all screen

### ISSUE THREAD

**Issue by snowiceyeah** [ControlzEx/ControlzEx#78]
Before run I set titlebar invisible and all buttons invisible.
```
public MainWindow()
        {
            InitializeComponent();
            IgnoreTaskbarOnMaximize = false;
            ShowMaxRestoreButton = false;
            ShowMinButton = false;
            ShowTitleBar = false;
            ShowCloseButton = false;
        }
```
![image](https://user-images.githubusercontent.com/43429385/58296260-94ece480-7e05-11e9-880a-e23e4cd38b53.png)
Then I set them back, the window can not fill the screen
 ```
private void Button_Click(object sender, RoutedEventArgs e)
        {
            ShowMaxRestoreButton = true;
            ShowMinButton = false;
            ShowMinButton = true;
            ShowTitleBar = true;
            ShowCloseButton = true;
        }
```
![image](https://user-images.githubusercontent.com/43429385/58296295-bd74de80-7e05-11e9-8b45-5f5c518bbd83.png)

**Comment by punker76** [comment]
@snowiceyeah I found the problem, but it's not in ControlzEx, it's a problem in MahApps. I will fix this there. Currently it's not possible to transfer this issue to MahApps...

**Comment by punker76** [comment]
@snowiceyeah https://github.com/MahApps/MahApps.Metro/issues/3506

**Comment by punker76** [comment]
@snowiceyeah this has now been fixed, and it will be released in the next version of ControlzEx v4.0
